app/components/description.hbs

```diff
@@ -1,18 +1,24 @@
 <section class='description'>
   <div class='description__heading'>
-    <img class='description__image' src='assets/images/question-mark.png' />
+    <img
+      class='description__image'
+      src='assets/images/question-mark.png'
+      alt='Question Mark Background'
+    />
     <h2 class='description__title'>
-      <span class="title__question">What is</span>
+      <span class='title__question'>What is</span>
       <span class='title__highlight'>Real Dev Squad</span>
     </h2>
   </div>
   <p class='description__content'>
-    Real Dev Squad is an online non-profit open source free fun community for people in tech, mainly
-    developers, designers, college students, or product managers, to come, learn and contribute
-    towards building a platform for our community, that helps upskill everyone.
+    Real Dev Squad is an online non-profit open source free fun community for
```

Comment:
I think we had a constants file 🤔 . Can it be part of it?

Reply:
Putting the description in constants is I think oversimplification. I have only put that data in constants which are mapped.